src/elona/ui.cpp

```diff
@@ -1469,15 +1490,16 @@ void render_hud()
     render_status_ailments();
 
     // Date label
-    render_date_label();
+    if (!g_config.digital_clock())
+        render_date_label();
 
     // Buffs
     gmode(2, 180);
     render_buffs();
     gmode(2);
 
     // Clock
-    render_clock();
```

Comment:
The function `render_clock` should be renamed to `render_analogue_clock()` because there are 2 kinds of clocks now.